### DISCUSSION TITLE
Process community rewards cycle: 12/16/23 - 1/12/24

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#930] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on our newly refined measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):

```
# NOTE ASA Stats Community rewards 12/16/23 - 1/12/24
AlgoRhythMatic 0.07 damo
Algoworm 0.6 damo
dragmz 0.07 damo
grzracz 0.14 damo
kerrilija 1.49 damo
leaf  0.03 damo
Miles 0.12 damo
Mocha 0.12 damo
moloch10 0.12 damo
RandomTask  0.03 damo
SCN9A 0.37 damo
UncleDooom 0.03 damo
TOTAL 3.19 damo
```
Please review and comment if there are any questions or concerns. The rewards noted above for this period use newly-refined rewards values discussed here: https://github.com/asastats/channel/discussions/405

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by ipaleka** [comment]
Oops, I forgot that the cycle ended, can you please add this one too https://github.com/asastats/channel/issues/843 ?

**Reply by AlgoRhythMatic** [reply]
Sure, I can add it now

Edit: as I interpreted it, I added Algoworm [F2], moloch [F2+F1], and kerrilija [F1].  Is this correct?  Should this now be finalized as archived?

**Reply by ipaleka** [reply]
> Edit: as I interpreted it, I added Algoworm [F2], moloch [F2+F1], and kerrilija [F1]. Is this correct? Should this now be finalized as archived?

We need to make this clearer, these collections/NFTs monthly issues - I've been adding marks in parentheses in related comments, so:

Algoworm [F2], moloch [F2], Mocha [F1], and kerrilija [F1]

I guess it'd be the best that I just before closing  such an issue simply put the marks in the body's first line...

**Reply by AlgoRhythMatic** [reply]
Ah, I think in this case it was just my old eyes scanning too fast and matching two mo* as moloch 🤣

**Reply by AlgoRhythMatic** [reply]
Ok, the list is updated!
